### PR TITLE
Update porn

### PR DESCRIPTION
**Pull request by chirag127**
Many NSFW domains which are not blocked are added.

### CODE REVIEW

**Review by glaciers7506 — APPROVED** [review]

File: porn [hunk 1]
```diff
@@ -362,6 +368,7 @@
 0.0.0.0 www.zetton-av.com
 0.0.0.0 www.zeus-web.net
 0.0.0.0 www2.giga-jp.com
+0.0.0.0 xhamster12.desi
```
Comment (spirillen) [inline porn:371]: curl -I 'http://xhamster12.desi/'
HTTP/1.1 301 Moved Permanently
Server: nginx
Date: Tue, 20 Jul 2021 13:25:00 GMT
Content-Type: text/html; charset=UTF-8
Connection: keep-alive
Set-Cookie: stats_id=214439; expires=Tue, 27-Jul-2021 13:25:00 GMT; Max-Age=604800; path=/; domain=.xhamster12.desi; secure; SameSite=None
Srv: main137
Cache-Control: no-cache, no-store, must-revalidate
ETag: 8e7b94a4b872654ec3a09d7f495dee94
Strict-Transport-Security: max-age=86400; preload
Vary: User-Agent
Location: https://xhamster.com
X-Frame-Options: SAMEORIGIN

File: porn [hunk 2]
```diff
@@ -301,6 +306,7 @@
 0.0.0.0 www.pierrot-net.jp
 0.0.0.0 www.pinknotora.net
 0.0.0.0 www.plum-web.com
+0.0.0.0 www.porn00.tv
```
Comment (spirillen) [inline porn:309]: Not enough records... See https://mypdns.org/my-privacy-dns/porn-records/-/issues/112

File: porn [hunk 3]
```diff
@@ -119,9 +122,11 @@
 0.0.0.0 pornjapanese.pro
 0.0.0.0 pornocuanimale.online
 0.0.0.0 pornoeggs.com
+0.0.0.0 porntn.com
```
Comment (spirillen) [inline porn:125]: Missing at least 2 records... See: https://mypdns.org/my-privacy-dns/porn-records/-/issues/110

File: porn [hunk 4]
```diff
@@ -55,6 +57,7 @@
 0.0.0.0 hentaihand.com
 0.0.0.0 hentaipornpics.net
 0.0.0.0 hipup.net
+0.0.0.0 hitbdsm.com
```
Comment (spirillen) [inline porn:60]: Missing at least 4 records, See: https://mypdns.org/my-privacy-dns/porn-records/-/issues/109

File: porn [hunk 5]
```diff
@@ -3,6 +3,8 @@
 0.0.0.0 ac-pink.net
 0.0.0.0 actionjav.com
 0.0.0.0 adult-gazou.me
+0.0.0.0 adult.noodlemagazine.com
+0.0.0.0 adultsearch.com
```
Comment (spirillen) [inline porn:7]: Missing a lot of subdomains.... See: https://mypdns.org/my-privacy-dns/porn-records/-/issues/108

**Review by spirillen — CHANGES_REQUESTED** [review]

File: porn (hunk 1 above)
Comment (spirillen) [reply porn:371]: This means it should not be in the list

**Comment by chirag127** [comment]
Well my reason to create the PR was that this list is used in the nextdns. But seems like now these domains are Already blocked by nextdns.

**Comment by spirillen** [comment]
Yep @nextdns have chosen to make yet another anti-porn project, rather than collaborate with existing lists.. 

https://github.com/nextdns/metadata/issues/569#issuecomment-860041895

**Comment by chirag127** [comment]
Hmm... Till yet they are combining many blocklist. And I think these domains Just got added to one of the included lists.

**Comment by spirillen** [comment]
Yep, they are all coming from https://mypdns.org/my-privacy-dns/porn-records via https://github.com/Clefspeare13/pornhosts

**Comment by Rexadev** [comment]
> Yep @nextdns have chosen to make yet another anti-porn project, rather than collaborate with existing lists..

Ask them to add it to json and give credit and pay hosting and other stuff
NextDNS is using mypdns.org but doesn't show in json. After I pull request 171gifs you added it to mypdns and they blocked it. Only blocklist nextdns have is PornAway (1 is notupdated other 2 are combined into the 4th one) .and it doesn't have 171gifs
I don't know how Clefspeare13/pornhosts is blocked

**Comment by spirillen** [comment]
> I don't know how Clefspeare13/pornhosts is blocked

Hmm actually a good question. I haven't though of how the movement (Do to take down) of that repo infected this repo, but I would as such assume they have upgraded there import path to match the new location.

**Pull request closed without merging**